compiler/compile/VirtualGuard.cpp

```diff
@@ -196,6 +196,8 @@ TR_VirtualGuard::createVftGuard
    aconstNode->setInlinedSiteIndex(calleeIndex);
    aconstNode->setByteCodeIndex(0);
 
+   comp->verifySymbolHasBeenValidated(static_cast<void *>(thisClass));
+
```

Comment:
Is there something particular about VFT (class) guards that make them important to handle but not (say) method guards ?

Reply:
This is some debug code that doesn't need to exist any more; I'll clean this stuff up.